lte/gateway/docker/docker-compose.yaml

```diff
@@ -94,45 +110,78 @@ services:
   eventd:
     <<: *pyservice
     container_name: eventd
+    healthcheck:
+      test: ["CMD", "nc", "-zv", "localhost", "50075"]
+      timeout: "4s"
+      retries: 3
     command: /usr/bin/env python3 -m magma.eventd.main
 
   health:
     <<: *pyservice
     container_name: health
+    healthcheck:
+      test: ["CMD", "nc", "-zv", "localhost", "50080"]
+      timeout: "4s"
+      retries: 3
     # Needed in order to enable/disable ICMP
     privileged: true
     command: /usr/bin/env python3 -m magma.health.main
 
   monitord:
     <<: *pyservice
     container_name: monitord
+    healthcheck:
+      test: ["CMD", "nc", "-zv", "localhost", "50076"]
+      timeout: "4s"
+      retries: 3
     command: /usr/bin/env python3 -m magma.monitord.main
 
   redirectd:
     <<: *pyservice
     container_name: redirectd
+    healthcheck:
+      test: ["CMD", "nc", "-zv", "localhost", "50071"]
+      timeout: "4s"
+      retries: 3
     command: /usr/bin/env python3 -m magma.redirectd.main
 
   smsd:
     <<: *pyservice
     container_name: smsd
+    healthcheck:
+      test: ["CMD", "nc", "-zv", "localhost", "50078"]
+      timeout: "4s"
+      retries: 3
     command: /usr/bin/env python3 -m magma.smsd.main
 
   control_proxy:
     <<: *pyservice
     container_name: control_proxy
+    healthcheck:
+      test: ["CMD", "nc", "-zv", "localhost", "8443"]
+      timeout: "4s"
+      retries: 3
     command: >
       sh -c "/usr/local/bin/generate_nghttpx_config.py && sed -i 's/errorlog-syslog=yes/errorlog-syslog=no/' /var/opt/magma/tmp/nghttpx.conf &&\
              /usr/bin/env nghttpx --conf /var/opt/magma/tmp/nghttpx.conf /var/opt/magma/certs/controller.key /var/opt/magma/certs/controller.crt"
 
   ctraced:
     <<: *pyservice
     container_name: ctraced
+    healthcheck:
+      test: ["CMD", "nc", "-zv", "localhost", "50079"]
```

Comment:
Is there a way to get this port value from the service registry config? if not, maybe we should add a note to change the value when one config is modified?

Reply:
Created a [new issue](https://github.com/magma/magma/issues/10578) to track this. Current thinking is to create a docker-compose template file that will be filled out runtime based on previously set values.